src/doc-templates/index.md.jinja2

```diff
@@ -0,0 +1,38 @@
+# {{ schema.name }}
+
+{{ schema.description }}
+
+URI: {{ schema.id }}
+
+
+## Classes
+
+| Class | Description |
+| --- | --- |
+{% for u, v in gen.class_hierarchy_as_tuples() -%}
+| {{ "&nbsp;"|safe*u*4 }} {{ gen.link(schemaview.get_class(v)) }} | {{ schemaview.get_class(v).description }} |
```

Comment:
do not include whitespace between `&nbsp` and the text, otherwise it will wrap for long texts